lib/wda/xcodebuild.js

```diff
@@ -162,7 +168,7 @@ class XcodeBuild {
       let [buildCmd, testCmd] = this.useSimpleBuildTest ? ['build', 'test'] : ['build-for-testing', 'test-without-building'];
       if (buildOnly) {
         args = [buildCmd];
-      } else if (this.usePrebuiltWDA) {
+      } else if (this.usePrebuiltWDA || this.useXCTESTRUNFile) {
```

Comment:
shouldn't we let this file to be built if it does not exist and WDA build is executed for the first time?

Reply:
i think we can do that in separate PR , as more advanced. Here we expect people already build it and keep it in `bootstrapPath` folder